clusterloader2/cmd/clusterloader.go

```diff
@@ -159,6 +161,20 @@ func main() {
 		glog.Fatalf("Parsing flags error: %v", errList.String())
 	}
 
+	// TODO: Replce glog with klog.
```

Comment:
NIT: A good TODO should be  associated with some issue or at least assigned to someone. 
An issue is preferable as usually it provides all the details for someone else to take over in case you won't be able to finish / do it yourself.

Could you create an issue and associate it with this TODO?

Reply:
Done.